chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -240,6 +245,7 @@
             variables(
                       :admin_node_ip => admin_ip,
                       :web_port => web_port,
+                      :packages => packages,
```

Comment:
Forget this and write ```packages = node[:provisioner][:packages][os] || []``` to assign it to the template resource of crowbar_register as well.